features/include/pcl/features/spin_image.h

```diff
@@ -258,8 +258,8 @@ namespace pcl
         * \param[in] index the index of the reference point in the input cloud
         * \return estimated spin-image (or its variant) as a matrix
         */
-      Eigen::ArrayXXd 
-      computeSiForPoint (int index) const;
+      template <int Storage = Eigen::ColMajor> auto
```

Comment:
What is the reason to use trailing return type here?

Reply:
The type is a bit too long and I was planning on adding something like `template <Storage> auto /*Eigen::ArrayXXd<Storage>*/` for readability if this worked.